### PR TITLE
Login/signup backend frontend link

### PR DESCRIPTION
**Pull request by ez4papiz1**
-Fixed session storage
-Linked login and signup backend and frontend
-added error handling on login and signup pages for the server not to crash when errors occur
-now can store new users in the database through the signup route
-assign unique 5 digit id to new users

### CODE REVIEW

File: DiscordCode/BackEnd/routes/login.js
Comment (Eneshjakhar) [inline DiscordCode/BackEnd/routes/login.js:1]: looks good to me! approve changes.

**Review by Eneshjakhar — COMMENTED** [review]
Looks good to me, might require one slight change!

File: DiscordCode/BackEnd/routes/signup.js [hunk 1]
```diff
@@ -1,26 +1,55 @@
...
-    res.render('signup');
+router.use(session({
+    secret: 'secretkey',
+    resave: false,
+    saveUninitialized: false,
+}));
+
+function generateUid() {
+    return Math.floor(100000 + Math.random() * 900000);
+}
```
[the remark this answers is not in the document]
Comment (Lunilux24) [reply DiscordCode/BackEnd/routes/signup.js:24]: I believe line 32 is what is checking for that.

**Review by Lunilux24 — APPROVED** [review]
All is good. Will merge now.

**Pull request merged**